cmd/jaeger/internal/integration/integration.go

```diff
@@ -4,12 +4,14 @@
 package integration
 
 import (
+	"fmt"
 	"io"
 	"os"
 	"os/exec"
 	"testing"
 
 	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
```

Comment:
not `gopkg.in/yaml.v3`?

Reply:
this returns false when using v3 but works fine with v2
```go
service, ok := config["service"].(map[interface{}]interface{})
```

Reply:
odd. Perhaps instead of doing conditional cast and checking OK we could do `require.IsType()` first, and if the types do not match it should print the actual type

Reply:
it worked. the type was `map[string]interface{}`